contents/docs/surveys/troubleshooting.mdx

```diff
@@ -54,3 +54,14 @@ You can call `getActiveMatchingSurveys` in your browser console to see which sur
 posthog.getActiveMatchingSurveys(surveys => console.log(surveys))
 ```
 
+## How are surveys priced?
+
+Surveys are priced based on the [number of responses](/pricing?product=surveys). However, because surveys is a tool that utilizes other parts of PostHog, such as events and feature flags, it's important to highlight the underlying costs too.
+
+Whenever a response is sent, it's stored as an event in PostHog, along with survey "shown" and "dismissed" events. Events are free up to the 1st million per month, and then $0.00031 after that. This should be negligible unless you're sending surveys to millions of users a month.
+
+Whenever user properties targeting or linked flags are used in a survey, feature flags are used underneath the hood. Feature flags are free up to the 1st million requests per month, and then $0.0001 after that.
```

Comment:
I did not mention "surveys flags are free if you don't have any other flags" because there isn't a losing point for the customer in that scenario that they need to be aware of